modules/images/auto-sizes/readme.txt

```diff
@@ -0,0 +1,53 @@
+=== Auto Sizes for Lazy-loaded Images ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.4
+Tested up to:      6.4
+Requires PHP:      7.0
+Stable tag:        1.0.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, images, auto sizes
+
+Adds support for automatically calculating the sizes attribute for lazy-loaded images.
+
+== Description ==
+
+This plugin implements the HTML spec for auto sizes for lazy-loaded images.
```

Comment:
Should we maybe include a mention of `sizes="auto"` here or elsewhere? While "Auto sizes" is a decent name, there's a good chance a developer will have a better idea of what this does when seeing `sizes="auto"`, as only that clarifies it's about the "sizes" _attribute_. "Auto sizes" could also have to do with e.g. the generated image sizes or other things.

I don't want to get hung up on that, but I think at least one mention of the attribute would be helpful in clarifying. For instance:
```suggestion
This plugin implements the HTML spec for `sizes="auto"` for lazy-loaded images.
```

Reply:
I'm happy to go with this. Thanks!